### PR TITLE
Package Order -> master

### PR DESCRIPTION
**Pull request by YHerm**

### CODE REVIEW

File: src/main/java/frc/joysticks/Axis.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package frc.utils.joysticks;
+package frc.joysticks;
```
Comment (hillelv) [inline src/main/java/frc/joysticks/Axis.java:1]: Why is joysticks outside of utils?

File: src/main/java/frc/JoysticksBindings.java [hunk 2]
```diff
@@ -1,7 +1,8 @@
-package frc.robot;
+package frc;
```
Comment (hillelv) [inline src/main/java/frc/JoysticksBindings.java:1]: Why is bindings outside of joysticks?

File: src/main/java/frc/RobotManager.java [hunk 3]
```diff
@@ -2,15 +2,13 @@
 // Open Source Software; you can modify and/or share it under the terms of
 // the WPILib BSD license file in the root directory of this project.
 
-package frc.robot;
+package frc;
```
Comment (hillelv) [inline src/main/java/frc/RobotManager.java:5]: Why is robot manager outside of robot?

File: src/main/java/frc/robot/RobotConstants.java
Comment (hillelv) [inline src/main/java/frc/robot/RobotConstants.java:1]: What other constants can be here?

**Review by tnrz135 — APPROVED** [review]

**Review by YoniKiriaty — APPROVED** [review]

**Review by drorta — APPROVED** [review]

**Pull request merged**